src/components/Profile/components/PersonalInfoList/index.js

```diff
@@ -485,6 +475,25 @@ const PersonalInfoList = ({
           </List>
         </CardContent>
       </Card>
+      {/* alumni update form */}
+      <GordonDialogBox
+        open={openAlumniUpdateForm}
+        title="Alumni Update Form"
+        fullWidth
+        maxWidth="lg"
+        cancelButtonClicked={() => {
+          setOpenAlumniUpdateForm(false);
+        }}
+        cancelButtonName="cancel"
+        className={styles.alumni_update_form}
+      >
+        <AlumniUpdateForm
+          profile={profile}
+          closeWithSnackbar={(status) => {
+            handleAlumniUpdateForm(status);
+          }}
+        />
+      </GordonDialogBox>
```

Comment:
It would probably be cleaner to have the GordonDialogBox encapsulated inside the AlumniUpdateForm. Then you just pass `profile`, `open`, `onClose`, and `createSnackbar` to `AlumniUpdateForm` and it handles displaying the dialog. This would be especially nice because then you could have the `Update` button as the primary action button of the dialog, right next to the `Cancel` button.